.rubocop_todo.yml

```diff
@@ -122,14 +121,8 @@ Rails/TimeZone:
 # SupportedStyles: always, conditionals
 Style/AndOr:
   Exclude:
-    - 'config/unicorn.rb'
     - 'lib/tasks/growstuff.rake'
 
-# Offense count: 1
-Style/AsciiComments:
```

Comment:
``` #   config.locales = {:en => 'English', :es => 'Español'}``` upsets things, silly check :/

Maybe chuck this back in or delete commented code?